packages/components/tree-v2/src/tree-node-content.vue

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
+import { h, defineComponent, inject } from 'vue'
+import { treeNodeContentProps } from './defaults'
+import type { RootTreeType } from './tree.type'
+
+export default defineComponent({
+  name: 'ElTreeNodeContent',
+  props: treeNodeContentProps,
+  setup(props) {
+    const tree = inject<RootTreeType>('RootTree')
```

Comment:
We should independently save this symbol as `ROOT_TREE_INJECTION_KEY` or something. in `tree.ts`